spec/models/membership_application_spec.rb

```diff
@@ -107,14 +107,14 @@
 
     it 'validates the presence of first_name' do
       expect {
-        member_app.first_name = ''
+        user.first_name = ''
         member_app.save!
       }.to raise_exception(/#{I18n.t('activerecord.attributes.membership_application.first_name')} #{I18n.t('errors.messages.blank')}/)
     end
 
     it 'validates the presence of last_name' do
       expect {
-        member_app.last_name = ''
+        user.last_name = ''
         member_app.save!
       }.to raise_exception(/#{I18n.t('activerecord.attributes.membership_application.last_name')} #{I18n.t('errors.messages.blank')}/)
     end
```

Comment:
Not yet in this PR. At the moment we still need to make sure you cannot save a membership application without a first name and lastname.